src/accountsdb/index.zig

```diff
@@ -104,17 +105,24 @@ pub const AccountIndex = struct {
                     .{sig.utils.fmt.tryRealPath(index_dir, ".")},
                 );
 
+                // reviewer's note: looks like we leak this + is there a point in allocating?
```

Comment:
Probably has something to do with the managed data structures inside the AccountIndex data structure holding `std.mem.Allocator` references to it, meaning it needs to be a stable pointer.

I think I might have tried to refactor this at some point but gave up, can't quite recall though. Feel free to take a stab at it if you want to see if you have a better time of it.

Reply:
Alright, just going to leave this be. Removed the note as it isn't a leak.

ff4b8b296aba104cf0e2f7a8c70da1e9e69914cf